test/checks/mobile/css-orientation-lock.js

```diff
@@ -23,11 +21,11 @@ describe('css-orientation-lock tests', function() {
 		MEDIA_STYLE_ORIENTATION_WITHOUT_TRANSFORM:
 			'@media screen and (min-width: 1px) and (max-width: 2000px) and (orientation: portrait) { #mocha { color: red; } }',
 		MEDIA_STYLE_ORIENTATION_WITH_TRANSFORM_NOT_ROTATE:
-			'@media screen and (min-width: 1px) and (max-width: 3000px) and (orientation: landscape) { #mocha { transform: translateX(10px); } }',
+			'@media screen and (min-width: 1px) and (max-width: 3000px) and (orientation: landscape) { #mocha { transform: translateX(10px); -webkit-transform: translateX(10px); } }',
```

Comment:
Please add a test using `-ms-transform`.

Reply:
We found out Phantomjs doesn't support `-ms-transform` since it uses a webkit engine under the hood, so we can't test this but we can add support for it in the code.